src/Linux/Avalonia.LinuxFramebuffer/LinuxFramebufferPlatform.cs

```diff
@@ -81,20 +81,20 @@ internal static LinuxFramebufferLifetime Initialize(AppBuilder builder, IOutputB
         }
     }
 
-    class LinuxFramebufferLifetime : IControlledApplicationLifetime, ISingleViewApplicationLifetime
+    public class LinuxFramebufferLifetime : IControlledApplicationLifetime, ISingleViewApplicationLifetime
```

Comment:
Not sure if it needs to be public.
You can subscribe on Edit/Startup events from IControlledApplicationLifetime interface which is public.

And you should be able to start this lifetime from AppBuilder.Start.